shuup/utils/i18n.py

```diff
@@ -5,19 +5,32 @@
 #
 # This source code is licensed under the OSL-3.0 license found in the
 # LICENSE file in the root directory of this source tree.
+from functools import lru_cache
+
 import babel
 import babel.numbers
 from babel import UnknownLocaleError
 from babel.dates import format_datetime
 from babel.numbers import format_currency, format_decimal, parse_pattern
 from django.apps import apps
 from django.utils import translation
-from django.utils.lru_cache import lru_cache
 from django.utils.timezone import localtime
 from django.utils.translation import get_language
 from django.views.decorators.cache import cache_page
 
 
+def lang_lru_cache(func):
+    """Language Aware Least-recently-used cache decorator"""
+
+    @lru_cache()
```

Comment:
No need to call it here, `@lru_cache` is a bit cleaner.

Reply:
`@lru_cache` doesn't support that syntax :(